src/core-members-group/CMGRedemptionHandler.sol

```diff
@@ -107,7 +107,7 @@ contract CMGRedemptionHandler is CMGHandler, ICMGRedemptionHandler, CirclesTypes
 
             // When a collateral's balance falls below minimal tracking amount
             // we remove it from our active tracking lists
-            if (remainingBalance <= _minimalTrackingAmount) {
+            if (remainingBalance <= _minimalTrackingAmount && balances[i] != 0) {
```

Comment:
do we need to apply the same patch at L141?

rather, is the better correction not to index activeCollateral from 1 rather than 0 and in _removeActiveId I should have checked that `indexInActiveIds[i] != 0`?